compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1437,20 +1457,20 @@ impl Default for TargetOptions {
         TargetOptions {
             is_builtin: false,
             endian: Endian::Little,
-            c_int_width: "32".to_string(),
-            os: "none".to_string(),
-            env: String::new(),
-            abi: String::new(),
-            vendor: "unknown".to_string(),
+            c_int_width: "32".into(),
+            os: "none".into(),
+            env: Cow::from(""),
```

Comment:
```suggestion
            env: "".into(),
```
Style nit: the `"string".into()` seems more common for constructing cows, so maybe it's better to normalize to it.